infer/src/base/Utils.ml

```diff
@@ -221,7 +221,22 @@ let do_finally_swallow_timeout ~f ~finally =
 
 
 let with_file_in file ~f =
-  let ic = In_channel.create file in
```

Comment:
Could you add a comment explaining why this isn't enough?

Reply:
Done. I must say I don't quite remember _why_ we needed `O_SHARE_DELETE`, but at least I addressed your comment :angel:.